fastpy/fastpy/run/Runner.py

```diff
@@ -0,0 +1,99 @@
+import os
+import shutil
+import itertools
+from subprocess import call
+
+from fastpy.io.config import load_json_config, store_json_config
+from fastpy.utils import get_date_time_tag
+from common import DATA_DIR_PATH, PROJECT_ROOT_PATH
+
+# This mapping will be used to pass command line arguments to benchmark executable
+PARAM_TO_C_MAP = {'algorithm':  '-a',
+                  'obj_func':   '-o',
+                  'dimension':  '-d',
+                  'n_iter':     '-n',
+                  'population': '-p',
+                  'min_val':    '-y',
+                  'max_val':    '-z'}
+
+BENCHMARK_BIN_DIR = os.path.join(PROJECT_ROOT_PATH, '../benchmark')
+BENCHMARK_BIN = 'benchmark'
+
+TIMING_OUT_FILE = 'timings.csv'
```

Comment:
I would either also move these into `common.py`, or not have a `common` at all, since they are only used in this module anyways.

Reply:
They are used in the output parsing module as well. Such that the output parser always has the correct names and/or templates. This is not in this PR yet but it will make sense in the next PR.

Reply:
Ok never mind then.